ch_pipeline/processing/daily.py

```diff
@@ -364,56 +460,67 @@ class DailyProcessing(base.ProcessingType):
     default_params = {
         # Time range(s) to process
         "intervals": [
-            ## Priority days to reprocess
-            # Good ranges from rev_00
+            # Two short intervals either side of the caltime change, and one
+            # with the weird 4 baseline issue in order to have a few good test
+            # days at the very start
+            {"start": "CSD1878", "end": "CSD1879"},
+            {"start": "CSD1912", "end": "CSD1913"},
```

Comment:
Can I request to look at `CSD1960` here as well? It's the most egregious example of a day with a corrupted frequency at a single baseline that wasn't masked in rev_03. My work in progress on modifying `ThresholdVisWeightBaseline` should catch it, but if that doesn't make it into rev_05, it would be good to see whether some other rev_05 change catches it.

Reply:
In that case, I'd also like to request `CSD1983` be put up front. It also has vertical stripes at certain baselines in rev_03 (attached plot only shows 3-cyl baselines - notice the stripes towards the right-hand side), but I haven't easily been able to localize the cause to specific time samples or frequencies by visual inspection. I'm interested to see how it looks in rev_05.
<img width="686" alt="Screen Shot 2022-09-27 at 11 05 48 PM" src="https://user-images.githubusercontent.com/14794540/192678718-76ebc960-b66f-4dd3-bb7a-411ddd76a8e8.png">

Reply:
This should be in the config now.